infrastructure/helm-chart/charts/core/charts/upgrading/templates/0.29.0.yaml

```diff
@@ -131,7 +101,7 @@ spec:
         volumeMounts:
         - name: upgrading-scripts
           mountPath: /opt/provisioning
-      serviceAccountName: airy-upgrade
+      serviceAccountName: {{ .Values.serviceAccount }}
```

Comment:
We can use the serviceAccount created by the CLI at install time. Can be modified from the `values.yaml` file, for the upgrading chart.